schemas/module-graph.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
I dislike that this adds a new top level directory. Can this go into cli/schemas/?